backend/tests/airline.test.js

```diff
@@ -0,0 +1,75 @@
+const supertest = require('supertest');
+const mongoose = require('mongoose');
+const app = require('../app');
+const airlineService = require('../services/airlineService');
+
+const airline = {
+  id: new mongoose.Types.ObjectId().toString(),
+  company: 'test airline',
+  airlineCode: 'AR12',
+  description: 'This is Test airline description',
+  flightIds: '62b590510cf8debe7e279001',
+};
+
+describe('airline', () => {
+  describe('get airline route', () => {
+    describe('given user not authenticated', () => {
+      it('should return a 401', async () => {
+        const airlineId = '123';
+        await supertest(app).get(`/api/airlines/${airlineId}`).expect(401);
```

Comment:
Please try to visually split test prepares, test execution and results check (add empty lines between them)

Reply:
Yes, I'll try to modify prettier as well. This tests were all right? I have experience in frontend tests with jasmine, but not with jest